### ISSUE TITLE
Garmin logging the same distances for every shot

### ISSUE THREAD

**Issue by jonchen727** [travislang/gspro-garmin-connect-v2#6]
Weird problem I have when using E6 connect and GSPro with the garmin connect-v2. Screenshots below show that all shots show a distance of 145 carry and 155 total with the same deviations. However when you look at the map view of the shots individually they show correctly. Im curious that if connect-v2 sends data back to the garmin, and its sending a static carry and shot shape data. It looks similar to the test shot being sent.

![C3F28D32-42EA-41F9-B76A-902CC99B5B91](https://user-images.githubusercontent.com/29684743/208857143-bbe8a81b-6742-44ed-a295-343584e8b011.png)
![9C6402F1-D1C1-4B0E-8150-6FB5E2B815DD](https://user-images.githubusercontent.com/29684743/208857163-6eff0f48-9cbe-433a-af41-d3c6d4f428b4.png)

**Comment by jonchen727** [comment]
It looks like its pulling some of the data from the get_shot_complete_message, looking at the GSProConnectV1 api it looks like we can grab the club from that and potentially return a dynamic message back to the E6Connect client, also, it looks like everything in the details gets over written except for deviation and distance, I might play around with it and see what happens if you return empty ball details, since something is happening on the Garmin app where it actually shows garmin's calculated distances in the graphic view as well as the correct club and ball data except the distances are wrong, which is interesting since static ball and club data are returned after each shot.

**Comment by that0n3guy** [comment]
@jonchen727 When we were messing w/ this, we found out that the Garmin software needs some data returned to it.  Garmin app sends shot data, then sits and waits for the "calculated" shot data to be returned back.  

Instead of figuring out how to get calculated shot data from GSPro and sending it back to the garmin app, @travislang just set some hard coded numbers to send back to the garmin app (I did the same thing when I was messing with it).   That's why everything is the same data: https://github.com/travislang/gspro-garmin-connect-v2/blob/9a0133db4f0dbe98c345da96d0a79fc110a98fcd/src/helpers/simMessages.js#L39
